### ISSUE TITLE
Does the WEB category really only include static websites?

### ISSUE THREAD

**Issue by linkvt** [informatiCup/informatiCup2017#8]
Hi,

with the addition of the additional data sets (thanks btw) there is a discussion we can't find a definitive answer about the WEB category.

**Aspects**
- english task description says `used to host static ... websites`
- german task doesn't include only static sites
- it says `personal websites or blogs` - what about other web sites
- your WEB test data contains a non-static jekyll repo: https://github.com/ianli/elbowpatched-boilerplate
- [your example data](https://github.com/BloombergMedia/whatiscode) and the [additional test data](https://github.com/ProseMirror/website) contains multiple repositories of sourcecode of websites, which have to be compiled first but are classified as WEB (I read that the jury might have classified some differently)
- DEV doesn't explicitly specify web applications

In my opinion this repo of you https://github.com/whoisjuan/whoisjuan.github.io is a typical example for WEB, since it contains static files for a personal website.
I can agree that a repository of a jekyll site (like https://github.com/ianli/elbowpatched-boilerplate) can be included in WEB.
I can't 100% see why e.g. https://github.com/BloombergMedia/whatiscode should be counted as WEB - since it has to be built first. At the same time it is not really an application as it could be interpreted by using the DEV definition.

In summary we are not sure whether your definition of WEB matches your examples and might be too exclusive (only static and only personal sites and blogs) or should be interpreted more freely to include everything which can result in a website - which would be difficult since there could be almost no differences to a DEV repo.
Thanks in advance!

**Comment by jonico** [comment]
Hi @linkvt,

thanks for your question! You are addressing an important point regarding the category definitions. We had a hard time agreeing on the exact wording for the label definitions as admittedly, there is quite some overlap between categories. It is fair to say that most repositories probably fall into multiple categories, so it is more the question to which category they fit in best.
This is why we stressed so many times that the documentation of your approach will outweigh the gained precision / accuracy of your submission for our (non disclosed) evaluation data set which has been created by humans.

https://github.com/BloombergMedia/whatiscode had been classified by multiple people to rather fit into WEB as primary category as opposed to DEV. The reason why we included the terms static and personal blog post into the WEB category description was to better differentiate it from DOC (content), EDU (content) and DEV repos that produce a web site which is using sophisticated backend logic to deliver its content dynamically for every request. Web sites produced by Jekyll are often static (as Jekyll does not run during an actual page request), as long as they do not include call backs to an application server that implements backend logic.

Hope that helps a bit. Good luck and merry Christmas!
  
![seasons-greetings](https://cloud.githubusercontent.com/assets/1872314/21424408/ffb11e2e-c842-11e6-8c67-94f0795ea5e6.png)

**Comment by linkvt** [comment]
Thanks a lot for your quick response and the explanation of your decision process, we now have a better understanding about the WEB category.
Merry Christmas!

**Comment by jonico** [comment]
Reopening this for transparency
